Automattic/jetpack
# Photon: get rid of jQuery dependency

Fixes #11181

#### Changes proposed in this Pull Request:


* replace jQuery dependency with vanilla javascript in Photon module

#### Testing instructions:

* Make sure that Photon module is enabled 
* Load a post with some images in it. Make sure that all `photon` hosted images have `width` & `height` CSS properties and HTML attributes. This can be checked (at least) in 2 ways:
- run this code snipped on in browser console: `document.querySelectorAll("img[src*='wp.com']").forEach( e => console.log('CSS value:', e.width, 'HTML attr:', e.getAttribute('width')))`. you should see pairs of same numbers. 
- Inspect image element in Chrome DevTools, under `Elements` tab select `Properties` tab in right column and inspect first node(`img.wp-image-*`) for `width` & `attributes > width` values
* Test with Lazy Load module enabled, with some images not loaded right away. 

#### Proposed changelog entry for your changes:


* Photon: replace jQuery dependency with vanilla javascript
